docs/source/includes/provisioning.rst

```diff
@@ -39,3 +39,14 @@ A subset of the hosts in a PinFile can be provisioned by listing each of them at
     Target              Run ID  uHash       Exit Code
     -------------------------------------------------
     dummy-new           83      a18e9a	            0
+
+
+
+Preview Feature:
+
+linchpin up and destroy includes --use-shell parameter which makes linchpin run as a subprocess rather than ansible api call
```

Comment:
Given what we discussed yesterday about modularization in the linchpin devel meeting, should we make this a flag?  If we make it the default and move to ansible galaxy later, it won't be a breaking change.  This implementation would.

Reply:
@14rcole Currently --use-shell is a flag in click module and is only enabled when used. If not used in command line it would take the ansible API route by default.
I think I might have misquoted in the documentation. 
If --use-shell is used linchpin runs ansible as a subprocess. 
Even if we move towards ansible-galaxy am not sure why would it break.

Reply:
If we move toward ansible galaxy, wouldn't be have to use a subprocess?  if so, using a subprocess should be the default

Reply:
I totally agree with you. Subprocess should be default, However there are many features we might have to make sure we implement which might not be in time for 1.8 release. 
That's the reason we are pushing out the preview with a flag. 
Enduser can default it to subprocess through linchpin.conf if they would like to.